package.json

```diff
@@ -104,17 +105,18 @@
     "check-licenses": "wp-scripts check-licenses --production",
     "deploy": "grunt deploy",
     "dev": "wp-scripts start",
+    "diff:develop": "git diff develop --diff-filter=d --name-only",
     "env:start": "./bin/local-env/start.sh",
     "env:stop": "./bin/local-env/stop.sh",
     "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:css": "wp-scripts lint-style",
+    "lint:css": "npm run diff:develop | grep -E 'css$' | xargs wp-scripts lint-style",
```

Comment:
`grep -E 'css$'` is to accomodate possible `.scss` files, though I don't think there are any now